xbmc/interfaces/legacy/Control.cpp

```diff
@@ -150,7 +150,7 @@ namespace XBMCAddon
     {
       if (!pGUIControl) return NULL;
 
-      LOCKGUI;
+      XBMCAddonUtils::GuiLock(languageHook);
```

Comment:
i noticed that a couple of python addons has this problem that gets kodi to lockup or crashing!